package.json

```diff
@@ -27,7 +27,8 @@
     "test:e2e": "cross-env NODE_ENV=development CI=true vitest e2e",
     "test:ui": "cross-env NODE_ENV=development CI=true vitest --exclude e2e --ui --coverage.enabled",
     "typecheck": "turbo typecheck",
-    "with-env": "dotenv -e .env --"
+    "with-env": "dotenv -e .env --",
+    "update:readme": "tsx ./scripts/update-integration-list.mts"
```

Comment:
Maybe we should change the name to something like `scripts:update-readme-integrations`, also because we might want to add other things that need updates in the readme. Or do you think we would just combine them then?